src/main/java/io/kestra/plugin/gemini/AbstractGemini.java

```diff
@@ -21,6 +23,25 @@
 @EqualsAndHashCode
 @Getter
 @NoArgsConstructor
+@Plugin(
```

Comment:
You have to remove this file from the changes and add the `metrics` block in the actual tasks (with existing `@Plugin` annotation) where the `AbstractGemini.sendMetrics` method is called so: 

<img width="1207" height="240" alt="Image" src="https://github.com/user-attachments/assets/677aebab-b188-4a1f-879c-a5d8b43620b0" />